internal/kubernetes/autoscaler.go

```diff
@@ -74,4 +74,37 @@ func scaleUpDeployment(item v1.Deployment, namespace string, clientDeployment v1
 	wg.Done()
 }
 
+func ScalingEngine() {
+	log.Info().Msgf("Starting Scaling Engine")
+	clientDeployment := clientSet.AppsV1().Deployments(config.Namespace)
+	labelSelector := metav1.ListOptions{
+		LabelSelector: fmt.Sprintf("%s=%s", config.LabelProxlessEnabled, "true"),
+	}
+	for true {
+		deploys, err := clientDeployment.List(labelSelector)
+		if err != nil {
+			log.Error().Err(err).Msgf("Could not list deployments with label %s=true in namespace %s", config.LabelProxlessSvc, config.Namespace)
+			// don't do anything else, we don't wanna kill the proxy
+		} else {
+			for _, item := range deploys.Items {
+				label := item.Labels[config.LabelProxlessSvc]
+				if route, err := store.GetRouteByLabel(label); err != nil {
```

Comment:
if you use `go fmt`, you'll see lines shouldn't go that far out in the std of go. as well.